docker_plugin/utils.py

```diff
@@ -253,9 +274,9 @@ def get_params(supported_params):
 
     d = {}
 
-    for key in ctx.node.properties['params'].keys():
```

Comment:
should probably raise an error when a key is unsupported, otherwise the behavior might be seen as unexpected (imagine a user misspells an input)